docs/reference/analysis/tokenfilters/keep-types-tokenfilter.asciidoc

```diff
@@ -20,9 +20,12 @@ produce a variety of token types, including `<ALPHANUM>`, `<HANGUL>`, and
 <<analysis-lowercase-tokenizer,`lowercase`>> tokenizer, only produce the `word`
 token type.
 
-Certain token filters can also add token types. For example, the 
+Certain token filters can also add token types. For example, the
 <<analysis-synonym-tokenfilter,`synonym`>> filter can add the `<SYNONYM>` token
 type.
+
+Some tokenizers don't support this token filter, for example keyword, simple_pattern, and
+simple_pattern_split tokenizers.
```

Comment:
I would add something like "as they don't support setting the token type attribute."

Reply:
Thanks @javanna so entire sentence would be like 
```
Some tokenizers don't support this token filter, for example keyword, simple_pattern, and
simple_pattern_split tokenizers, as they don't support setting the token type attribute.
```
let me know if this sounds OK to you and I would make the required change.